app/code/Magento/Newsletter/Test/Mftf/ActionGroup/DisableConfirmNewsletterEnableRegistrationConfirmActionGroup.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+
+    <actionGroup name="DisableConfirmNewsletterEnableRegistrationConfirmActionGroup">
+
+        <amOnPage url="{{AdminCustomerConfigPage.url('#customer_create_account-link')}}" stepKey="goToCustomerConfig"/>
+        <waitForPageLoad stepKey="waitForCustomerAdminPageLoad"/>
+        <checkOption selector="{{AdminCustomerConfigSection.customerCreateAccountCheckbox}}" stepKey="checkConfirmCreateAccountCheckbox"/>
+        <click selector="{{AdminMainActionsSection.save}}" stepKey="saveCustomerConfig"/>
```

Comment:
It is good to scroll to top of the page before click save button. The same for second use in this AG